dataprofiler/profilers/json_decoder.py

```diff
@@ -3,12 +3,14 @@
 
```

Comment:
Curious if we can consolidate `_profiles`, `_compilers`, and `_options` into one grouping, cause there is a lot of duplicate code. If we can, would it make sense to do so?

Reply:
Either via one dict or a func that takes in the dict. I'm curious of the mypy....